src/page-modules/contact/ticket-control/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { PageText } from '@atb/translations';
+import { SectionCard } from '../components/section-card';
+import { FeeComplaintForm } from './complaint/feeComplaintForm';
+import { ContactForm, ContactFormSelector } from '../components/form-selector';
+import { useRouter } from 'next/router';
+import { renderSelectedPage } from '../utils';
+
+const contactForms: ContactForm[] = [
+  {
+    description: PageText.Contact.ticketControl.optionDescriptions.feeComplaint,
+    href: '/contact/billettkontroll-og-gebyr/klage',
+    page: <FeeComplaintForm />,
+  },
+  {
+    description: PageText.Contact.ticketControl.optionDescriptions.refund,
+    href: '/contact/billettkontroll-og-gebyr/utsette-betaling',
+    page: <div>Utsette betaling</div>,
+  },
+  {
+    description: PageText.Contact.ticketControl.optionDescriptions.feedback,
+    href: '/contact/billettkontroll-og-gebyr/tilbakemelding',
+    page: <div>Tilbakemelding</div>,
+  },
+];
+
+export type TicketControlAndFeeContentProps = { title: string };
+
+export const TicketControlAndFeeContent = (
+  props: TicketControlAndFeeContentProps,
+) => {
+  const router = useRouter();
+
+  return (
+    <div>
+      <SectionCard title={PageText.Contact.ticketControl.title}>
+        <ContactFormSelector contactForms={contactForms} />
+      </SectionCard>
+
+      {renderSelectedPage(contactForms, router.asPath)}
+    </div>
```

Comment:
It seems to me that we're reinventing the wheel here and making things more complicated than necessary. Using a layout with shallow navigation to the sub-pages could achieve the same outcome. We could simply have a list of links to the existing sub-pages. This approach would eliminate the need to keep the contactForms variable in this file updated with the correct paths, reducing the potential for future errors